Observer/KlaviyoUserObserver.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Klaviyo\Reclaim\Observer;
+
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Framework\Event\Observer;
+use \Magento\Authorization\Model\Acl\Role\Group as RoleGroup;
+
+class KlaviyoUserObserver implements ObserverInterface
+{
+    /**
+     * ManagerInterface
+     * 
+     * @var _messageManager
+     */
+    protected $_messageManager;
+
+    /**
+     * DataHelper
+     * 
+     * @var _dataHelper
+     */
+    protected $_dataHelper;
+
+    /**
+     * UserFactory
+     * 
+     * @var _userFactory
+     */
+    protected $_userFactory;
+
+    /**
+     * RoleCollectionFactory
+     * 
+     * @var RoleCollectionFactory
+     */
+    protected $_roleCollectionFactory;
+
+    const KLAVIYO_FIRST_NAME = 'klaviyo';
+    const KLAVIYO_LAST_NAME = 'klaviyo';
+    const KLAVIYO_ROLE_NAME = 'Klaviyo';
+    const DEFAULT_LOCALE = 'en_US';
+
+    /**
+     * Init
+     *
+     * @param MessageManager $_messageManager
+     * @param DataHelper $_dataHelper
+     * @param UserFactory $_userFactory
+     * @param RoleCollectionFactory $_roleCollectionFactory
+     */
+    public function __construct(
+        \Magento\Framework\Message\ManagerInterface $_messageManager,
```

Comment:
just some nit pick but could we alphabetize all these things? Ex. magento things then klaviyo things?